index.html

```diff
@@ -0,0 +1,459 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="./css/main.css" />
+    <link rel="icon" href="./favicon.ico" />
+    <title>Aziz Mejri - Portfolio</title>
+  </head>
+  <body>
+    <header>
+      <h1 id="logo">Aziz Mejri</h1>
+      <button type="button" class="menu-bars">
+        <i class="fas fa-bars"></i>
+      </button>
+      <nav id="mobile-nav">
+        <ul>
+          <li><a class="nav-link" href="#projects">Portoflio</a></li>
+          <li><a class="nav-link" href="#about-me">About</a></li>
+          <li>
+            <a class="nav-link" href="#contact"
+              >Contact <i class="far fa-envelope"></i
+            ></a>
+          </li>
+        </ul>
+      </nav>
```

Comment:
- [ ] Please add the close icon to your mobile nav bar as in the design. This will provide a clear way to users to close the nav bar.

![figma 1 z0](https://user-images.githubusercontent.com/99656687/210979938-021b1e33-347d-4ca0-9205-6f85fe1aae6a.png)

- [ ] [ OPTIONAL] Please consider adding a feature to make the menu disappear when a user taps anywhere within te menu. This improves the experience on small screen where the close icon may not be easily accesible.